index.html

```diff
@@ -524,6 +520,7 @@ <h2 class="form-heading">
             ></textarea>
             <small id="errorCheck" class="errorMessage"></small>
             <input
+              id="sub"
```

Comment:
[OPTIONAL] Well done overall 💯 I'm not sure if this can be fixed or how it can be fixed but the link provided to go back to your webpage is broken. 😞 👇🏽 

![Screenshot 2023-04-14 at 08 45 59](https://user-images.githubusercontent.com/45739963/232047306-4ced84df-f3af-43ce-aba1-488df5987d83.png)

PS: I tested on the live version 🚀

Reply:
I don't think it's fixable using that website